### ISSUE TITLE
[Request] ScriptSafe dark theme

### ISSUE THREAD

**Issue by qazip** [overdodactyl/ShadowFox#67]
Hello,

I'm here for another request!

* webextension name: ScriptSafe
* link to the webextension in the AMO store: https://addons.mozilla.org/en-US/firefox/addon/script-safe/ and https://github.com/andryou/scriptsafe
* what needs to be styled (settings in about:addons, popup etc.): I'd say both the settings in about:addons and the popup. The popup is more important.

This is a great alternative to NoScript for people who use different computers. It supports firefox sync, which is awesome. 

-->

**Comment by overdodactyl** [comment]
Hi @qazip, I'll get to this when I have a chance, but it admittedly probably won't be a top priority. 

It might be worth noting that I already have styles available for uBlock Origin and uMatrix (my personal recommendations for content/script blockers), as well as NoScript.  All of these have support for Firefox Sync.

**Comment by qazip** [comment]
@overdodactyl, I don't like umatrix because of its lack of auto-reload. A deal breaker for me. I was not aware that NoScript had support for firefox sync. Are you sure?

**Comment by overdodactyl** [comment]
>Are you sure?

Yep!

**Comment by qazip** [comment]
Yes, noscript does seem to support firefox sync, though not very well according to my tests. Still trying to find out when it syncs and when it does not.

Could you please add 

@import "userContent-files/webextension-tweaks/noscript.css" 

to the userContent_imports.css? It's missing there.

**Comment by overdodactyl** [comment]
Added! Sorry for the delay

**Comment by difool2nice** [comment]
hello, first great great theme and impressive work !
some issues with firefox 61 x64 windows 10 and the webext ublock origin ! check squares in options page are hidden
Keep up the good work and thumb up for this theme !

![2018-06-29_17h57_40](https://user-images.githubusercontent.com/25643255/42102501-5a54db6c-7bc6-11e8-90e4-516710b243ab.png)

![2018-06-29_17h57_52](https://user-images.githubusercontent.com/25643255/42102502-5c83e3f6-7bc6-11e8-9072-a4fd6ae9be4f.png)
